src/Components/RecsListTable/index.js

```diff
@@ -62,107 +64,118 @@ const RecsListTable = () => {
   }, [data, filters]);
 
   // constructs array of rows (from the initial data) checking currently applied filters
-  const buildFilteredRows = (allRows, filters) => {
-    return (
-      allRows
-        .filter((rule) => passFilters(rule, filters))
-        // TODO: replace with sortable rows
-        .sort((fst, snd) => (fst.total_risk < snd.total_risk ? 1 : -1))
-        .map((value, key) => [
-          {
-            isOpen: false,
-            rule: value,
-            cells: [
-              {
-                title: (
-                  <span key={key}>
-                    <Link
-                      key={key}
-                      // https://github.com/RedHatInsights/ocp-advisor-frontend/issues/29
-                      to={`/recommendations/${
-                        process.env.NODE_ENV === 'development'
-                          ? value.rule_id.replaceAll('.', '%2E')
-                          : value.rule_id
-                      }`}
-                    >
-                      {' '}
-                      {value?.description || value?.rule_id}{' '}
-                    </Link>
-                    <RuleLabels rule={value} />
-                  </span>
-                ),
-              },
-              {
-                title: value?.publish_date ? (
-                  <DateFormat
+  const buildFilteredRows = (allRows, filters, index) => {
```

Comment:
Is it expected that you keep some of the `buildFilteredRows` calls without the third parameter? I mean, here https://github.com/RedHatInsights/ocp-advisor-frontend/blob/9ead83983159bf292f726e51564bac6248393f2b/src/Components/RecsListTable/index.js#L63, the function does not provide `index`. What would happen when `index` is undefined?

Reply:
I would also provide a sorting direction parameter (or just a single `sortBy` object) and keep direction logic inside the function. Right now, you decide whether to reverse the rows here https://github.com/RedHatInsights/ocp-advisor-frontend/blob/9ead83983159bf292f726e51564bac6248393f2b/src/Components/RecsListTable/index.js#L313